### PR TITLE
add zc preprocessor

### PR DESCRIPTION
**Pull request by anporumb**

### CODE REVIEW

File: macro_utils_h_generator/macro_utils_h_generator.csproj [hunk 1]
```diff
@@ -31,6 +31,9 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
+  <PropertyGroup>
+    <SignAssembly>false</SignAssembly>
+  </PropertyGroup>
```
Comment (dcristoloveanu) [inline macro_utils_h_generator/macro_utils_h_generator.csproj:36]: Should be True

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

File: macro_utils_h_generator/macro_utils_h_generator.csproj (hunk 1 above)
Comment (anporumb) [reply macro_utils_h_generator/macro_utils_h_generator.csproj:36]: It is now...

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

**Pull request merged**